src/pages/GivePage/GiveRow.tsx

```diff
@@ -71,6 +71,35 @@ export const GiveRow = ({
   );
 };
 
+const MemberActivity = ({
+  member,
+  contributionsCount,
+}: {
+  member: Member;
+  contributionsCount: number;
+}) => {
+  return (
+    <>
+      {(member.pending_sent_gifts.length > 0 || contributionsCount > 0) && (
```

Comment:
if `(member.pending_sent_gifts.length > 0` is false, but they've made contributions, we will show Allocated, even though they have not allocated... right? @wingfeatherwave

Reply:
@crabsinger no it won't show.  there is another check on 85 and 91.  This parent check is just for the flex container.  Lemme rewrite it so the component doesn't define the gap, then the logic will be simpler.